src/plugins/mer/mersdkdetailswidget.ui

```diff
@@ -53,6 +53,9 @@
           <string>Virtual machine</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
+          <property name="formAlignment">
+           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
```

Comment:
At least according to Qt documentation, AlignLeading and AlignLeft are synonyms

Reply:
I noticed that too, but the same combination is used also elsewhere in this file. Should I remove it from the other lines too?

Reply:
I have no strong opinions on whether you should remove it from the other lines or not. But if you do remove it, you should do it in a separate commit.